test/board_test.rb

```diff
@@ -57,46 +106,99 @@ def test_determines_if_placement_coordinates_are_sequential_in_y_direction
     assert_equal false, board.y_coordinates_sequential?(["A1", "B2"])
   end
 
-  def test_determines_if_placement_coordinates_are_sequential_in_y_direction
-    board = Board.new
-    cruiser = Ship.new("Cruiser", 3)
-
-    assert_equal true, board.valid_placement?(cruiser, ["A1", "B1", "C1"])
-    assert_equal true, board.valid_placement?(cruiser, ["C2", "B2", "A2"])
-    assert_equal true, board.valid_placement?(cruiser, ["C1", "C2", "C3"])
-    assert_equal false, board.valid_placement?(cruiser, ["E1", "E2", "E3"])
-
-    assert_equal false, board.valid_placement?(cruiser, ["A1", "B2", "C3"])
-    assert_equal false, board.valid_placement?(cruiser, ["C2", "A2", "B2"])
-    assert_equal false, board.valid_placement?(cruiser, ["A1", "B2"])
-
-  end
 
   def test_it_places_ships_in_cells
     board = Board.new
     cruiser = Ship.new("Cruiser", 3)
-    #board.place(cruiser, ["A1", "A2", "A3"])
     board.place(cruiser, ["A1", "A2", "A3"])
-    #binding.pry
-    cell_A1 = board.cells["A1"]
-    cell_A2 = board.cells["A2"]
-    cell_A3 = board.cells["A3"]
-    cell_A4 = board.cells["A4"]
-#binding.pry
+
     assert_equal "Cruiser", board.cells["A1"].ship.name
     assert_equal "Cruiser", board.cells["A2"].ship.name
     assert_equal "Cruiser", board.cells["A3"].ship.name
     assert_nil board.cells["A4"].ship
     assert_nil board.cells["B1"].ship
   end
 
+  def test_it_identifies_cells_with_ships
```

Comment:
Nice work on this one, I think we both tested this but good to double up on the classes